packages/nimble-components/src/table-column/base/types.ts

```diff
@@ -15,6 +15,17 @@ export interface TableCellState<
     columnConfig: TColumnConfig;
 }
 
+export interface TableColumnWithPlaceholderColumnConfig {
```

Comment:
I added this interface to allow the anchor column to use the same group header element as the text column. It seemed like a waste to introduce a new element since we want the exact same behavior. The only hurdle was that the text group header element was strongly typed against `TableColumnTextColumnConfig`, even though it only used the `placeholder` property. The anchor column config also has `placeholder`, so by making both column configs extend this new interface, I could share the text column's group header.